[docs] Update operator docs to include parallel update annotation (#3407)
m3db/m3: site/content/operator/getting_started/update_cluster.md

```bash
kubectl annotate statefulset -l operator.m3db.io/cluster=my-cluster operator.m3db.io/update=enabled
```
## Parallel Pod Updates

> In larger M3DB clusters, the default of updating one pod at a time may be too slow. To update multiple pods in parallel, you should do the following:

Reviewer: ```suggestion
In larger M3 clusters, the default of updating one pod at a time may be too slow. To update multiple pods in parallel, you should do the following:
```

We're slowly trying to be more consistent in the way we refer to things…

1. Set `onDeleteUpdateStrategy` to `true` in your `M3DBCluster` CRD
2. Use the parallel update annotation when ready to update a cluster. For example:

> ```

Reviewer: ```suggestion
```shell
```

kubectl annotate statefulset my-cluster-rep0 operator.m3db.io/parallel-update=3
```

> In the example above, the operator will update three pods at a time, instead of one.

Reviewer: ```suggestion
In the example above, the operator updates three pods at a time, instead of one.
```
